presto-docs/src/main/sphinx/release/release-330.rst

```diff
@@ -0,0 +1,131 @@
+* Fix query failure for certain co-located joins when dynamic filtering is enabled. (:issue:`2685`)
+* Fix failure in ``SHOW CATALOGS`` when the user does not have permissions to see any catalogs. (:issue:`2593`)
+* Improve query performance for some join queries when :doc:`/optimizer/cost-based-optimizations`
+  are turned on. (:issue:`2722`)
+* Prevent uneven distribution of data that could occur when writing data
+  if redistribution or writer scaling is enabled. (:issue:`2788`)
+* Add support for ``CREATE VIEW`` with comment (:issue:`2557`)
+* Add support for all major geometry types to :func:`ST_Points`. (:issue:`2535`)
+* Add ``required_workers_count`` and ``required_workers_max_wait_time`` session properties
+  to control the number of workers that must be present in the cluster before the query
+  starts. (:issue:`2484`)
+* Add ``physical_input_bytes`` column to ``system.runtime.tasks`` table. (:issue:`2803`)
+* Verify that the target schema exists for the :doc:`/sql/use` statement. (:issue:`2764`)
+* Verify that the session catalog exists when executing :doc:`/sql/set-role`. (:issue:`2768`)
+
+
+Server Changes
+--------------
+
+* Require running on Java 11. This requirement may be temporarily relaxed by adding
+  ``-Dpresto-temporarily-allow-java8=true`` to the Presto :ref:`presto_jvm_config`.
+  This fallback will be removed in future versions of Presto after March 2020. (:issue:`2751`)
+* Add experimental support for running on Linux aarch64 (ARM64). (:issue:`2809`)
+
+Security Changes
+----------------
+
+* :ref:`principal-rules` are deprecated and will be removed in a future release.
+  These rules have been replaced with :doc:`/security/user-mapping`, which
+  specifies how a complex authentication user name is mapped to a simple
+  user name for Presto, and :ref:`impersonation-rules` which control the ability
+  of a user to impersonate another user. (:issue:`2215`)
+* A shared secret is now required when using :doc:`security/internal-communication`.  (:issue:`2202`)
+* Kerberos for :doc:`security/internal-communication` has been replaced with the new shared secret mechanism.
+  The ``internal-communication.kerberos.enabled`` and ``internal-communication.kerberos.use-canonical-hostname``
+  configuration properties must be removed. (:issue:`2202`)
+* When authentication is disabled, the Presto user may now be set using standard
+  HTTP basic authentication with an empty password.  (:issue:`2653`)
+
+Web UI Changes
+--------------
+
+* Display physical read time in detailed query view. (:issue:`2805`)
+
+JDBC Driver Changes
+-------------------
+
+* Fix a performance issue on JDK 11+ when connecting using HTTP/2. (:issue:`2633`)
+* Implement ``PreparedStatement.setTimestamp`` variant that takes a ``Calendar``. (:issue:`2732`)
+* Add ``roles`` property that allows to configure authorization roles to be used for catalogs. (:issue:`2780`)
```

Comment:
```
* Add ``roles`` property for authorization roles to be used for catalogs.

Reply:
or even simpler

Add ``roles`` property for catalog authorization roles